pre-check/v1.x/check.sh

```diff
@@ -450,7 +450,7 @@ check_free_space()
                     continue
                 fi
 
-                stat_summary=$(curl -sk https://${node_ip}:${kubelet_port}/stats/summary -H "Authorization: Bearer ${token}" | jq .node.fs)
+                stat_summary=$(curl --noproxy '*' -sk https://${node_ip}:${kubelet_port}/stats/summary -H "Authorization: Bearer ${token}" | jq .node.fs)
```

Comment:
Can we change it to use `kubectl get --raw "/api/v1/nodes/${node_name}/proxy/stats/summary"`?